app/assets/stylesheets/pages/_home.scss

```diff
@@ -0,0 +1,197 @@
+.home {
+  @include clearfix();
+  @include breakpoint(min-width 900px) {
+    h2,
+    p {
+      text-align: left;
+      width: 100%;
+    }
+  }
+
+  background-color: $c-blue;
+  color: white;
+  min-height: 100vh;
+  overflow: hidden;
+  position: relative;
+}
+
+.home__header {
+  clear: both;
+  padding: rem(18) 0;
+}
+
+.home__logo {
+  @include fluid-size("font-size", 16px, 36px);
+  @include breakpoint(min-width 900px) {
+    margin: 0 0 0 rd(85, 1024);
+  }
+
+  color: white;
+  font: 600 $f-logo;
+  margin: 0 rd(20, 410);
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    border-bottom: 4px solid white;
+    border-radius: 2px;
+  }
+}
+
+.home__inner {
+  @include breakpoint(min-width 1024px) {
+    padding: 5vh 0 0 0;
+  }
+
+  @include clearfix();
+  margin: 0 auto;
+  max-width: 2040px;
+  position: relative;
+}
+
+.home__inner-section {
+  @include breakpoint(min-width 1440px) {
+    max-width: 700px;
+  }
+
+  @include breakpoint(min-width 1024px) {
+    margin: 0;
+  }
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  margin: 0 auto;
+  max-width: 600px;
+}
+
+.home__svg-container {
+  @include breakpoint(min-width 1024px) {
```

Comment:
if you're re-using these breakpoints a lot, the it's good to have them as variables:

`$breakpoint-medium: 1024px;`
`@include breakpoint(min-width $breakpoint-medium) {`